add-on/panels/css/slideshow.css

```diff
@@ -0,0 +1,179 @@
+@import url('http://fonts.googleapis.com/css?family=Open+Sans');
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+html {
+  font-size: 10px;
+  font-family: menu;
+  color: #fff;
+}
+
+body {
+  background: none;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  line-height: 2.4rem;  /* or 1.3rem original*/
+}
+.slideshow {
+
+}
+.slides {
+  display: block;
+  width: 100%;
+  height: 390px;
+  overflow: hidden;
+  white-space: nowrap;
+}
+.slide {
+  display: none;
+  height: 100%;
+  width:  100%;
+}
+.slide.slide--active {
+  display: block;
+}
+
+.control-panel {
+  height: 60px;
+  background: #fff;
+  width: 100%;
+}
+
+.controls {
+}
```

Comment:
When you move this section to the vendor based css file, please also remove the empty css blocks.